packages/playground/blueprints/src/lib/steps/install-asset.ts

```diff
@@ -75,6 +83,30 @@ export async function installAsset(
 
 		// Move asset folder to target path
 		const assetFolderPath = `${targetPath}/${assetFolderName}`;
+
+		// Handle the scenario when the asset is already installed.
+		if (await playground.fileExists(assetFolderPath)) {
+			if (!(await playground.isDir(assetFolderPath))) {
```

Comment:
Let's also check if there are some files there? Having empty directories caused issues in wp-now in the past and this might prevent issues like this.